Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml

```diff
@@ -177,6 +177,7 @@
                         android:id="@+id/settings_tracing_status_bluetooth"
                         gone="@{!settingsTracingState.isBluetoothCardVisible()}"
                         layout="@layout/include_tracing_status_card"
+                        android:layout_marginBottom="@dimen/spacing_small"
```

Comment:
I would increase the margin
![image](https://user-images.githubusercontent.com/64483219/114187649-8439af00-9948-11eb-885a-804041f4aebe.png)